blocks/releases/releases.js

```diff
@@ -0,0 +1,110 @@
+function mdToHTML(md) {
+  const toHTML = md
+    .replace(/^### (.*$)/gim, '<h3>$1</h3>') // h3 tag
+    .replace(/^## (.*$)/gim, '<h2>$1</h2>') // h2 tag
+    .replace(/^# (.*$)/gim, '<h1>$1</h1>') // h1 tag
+    .replace(/^\* (.*$)/gim, '<li>$1</li>') // li tag
+    .replace(/\*\*(.*)\*\*/gim, '<b>$1</b>') // bold text
+    .replace(/\*(.*)\*/gim, '<i>$1</i>') // italic text
+    .replace(/\[([^\]]+)\]\(([^)]+)\)/gim, '<a href="$2">$1</a>'); // links
+  return toHTML.trim(); // using trim method to remove whitespace
+}
+
+const displayNames = {
+  'helix-pipeline-service': 'Delivery Pipeline Service',
+  'helix-admin': 'Admin Service / API',
+  'helix-importer-ui': 'Content Importer Tooling',
+  'helix-cli': 'AEM Developer Command Line Interface',
+  'helix-sidekick-extension': 'AEM Authoring Sidekick',
```

Comment:
I would remove product branding and simplify:
```suggestion
  'helix-pipeline-service': 'Delivery Pipeline',
  'helix-admin': 'Admin API',
  'helix-importer-ui': 'Content Importer',
  'helix-cli': 'Command Line Interface',
  'helix-sidekick-extension': 'Sidekick Extension',
```